### PR TITLE
Add live streams to getindexes API method

### PR DESCRIPTION
**Pull request by 4phun**
Hi,

I would like to support Radio Stations in my client. This pull request adds them to the get_indexes API method. Please consider for inclusion in the next version.

Thank you

### CODE REVIEW

**Comment by lachlan-00** [comment]
Sounds good. I'll test and merge on Monday

File: lib/class/xml_data.class.php [hunk 1]
```diff
@@ -480,6 +480,14 @@ public static function indexes($objects, $object_type, $full_xml = true, $includ
                 case 'video':
                     $string .= self::videos($objects);
                     break;
+                case 'live_stream':
+                    $live_stream = new Live_Stream($object_id);
+                    $live_stream->format();
```
Comment (lachlan-00) [inline lib/class/xml_data.class.php:485]: don't think the format is needed as f_name is just scrubbed just finalising now.

**Comment by lachlan-00** [comment]
actually lets just keep the format

**Comment by 4phun** [comment]
Cool, thank you

**Comment by lachlan-00** [comment]
found a bug in the json while adding the json stuff so that's helped!

**Pull request merged**